src/utils/injectTreeNode.js

```diff
@@ -14,40 +16,46 @@ export default (
     style,
   } = {}
 ) =>
-  injectDefaults(({ tree, node, group, path, ...props }) => {
-    node = node || tree.getNode(path)
+  _.flow(
+    inject(({ theme = {} }) => ({
+      Loading: (theme.Loading || StripedLoader)(render, style),
```

Comment:
If we're generalizing this, we probably need a way for custom loaders to have access to `render`, otherwise it will just always show the loading indicator, right?

Reply:
I'm not sure I follow. They have access to render. It's being passed as the first argument to the loaders

Reply:
Yup, I was confused because `Loading` and `theme.Loading` are so similar. This does make me think we should reconsider the "loader" API to just be a component instead of a function though